sessions/css-basics/pink-box/css/styles.css

```
@@ -0,0 +1,25 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: system-ui;
+}
+
+/* put a selector here to select div elements (see index.html) */
+SELECTOR HERE {
+  font-family: cursive;
+  height: 150px;
+  width: 150px;
```

If the challenge is to make a box, would it make sense for them to set the height and width themselves? My concern is that sometimes, when code is already provided, it can get "skipped" over. 

```suggestion
  /* set the height and the width of the box to 150px */
```

---

great idea!